core/kazoo_data/src/kzs_attachments.erl

```diff
@@ -205,14 +211,22 @@ attachment_url(#{server := {App, Conn}}, DbName, DocId, AttachmentId, 'undefined
 attachment_url(_, DbName, DocId, AttachmentId, Handler, Options) ->
     {'proxy', {DbName, DocId, AttachmentId, [{'handler', Handler} | Options]}}.
 
--spec save_attachment_handler_error(kz_term:ne_binary(), kz_att_error:error()) -> 'ok'.
-save_attachment_handler_error(DbName, {'error', Reason, ExtendedError}) ->
-    NewValues = [{'reason', Reason} ,{'pvt_type', <<"attachment_handler_error">>}],
+-spec save_attachment_handler_error(att_map()
+                                   ,kz_term:ne_binary()
+                                   ,kz_att_error:error()
+                                   ) -> 'ok'.
+save_attachment_handler_error(#{'att_handler_id' := AttUUID}
```

Comment:
this will fail for `kzc_recording` & `kz_endpoint_recording`
use something like
```
save_attachment_handler_error(Map)
...
AttUUID = maps:get(att_handler_id, Map, <<"overriden">>),